servers/display_server.h

```diff
@@ -210,6 +210,10 @@ class DisplayServer : public Object {
 	virtual void tts_set_utterance_callback(TTSUtteranceEvent p_event, const Callable &p_callable);
 	virtual void tts_post_utterance_event(TTSUtteranceEvent p_event, int p_id, int p_pos = 0);
 
+	virtual bool is_dark_mode_supported() const { return false; };
+	virtual bool is_dark_mode() const { return false; };
+	virtual Color get_accent_color() const { return Color(); };
```

Comment:
Should there be a `has_accent_color` method to check if the OS supports it or is unset? Otherwise we have no way of knowing if the returned black color is the user's preference or the default when the OS doesn't support it or is unimplemented.

Reply:
As far as I know, Windows doesn't allow you to choose an accent color darker than `#333333` (even with the custom color field).

The documentation could mention that a pure black color indicates that the OS doesn't support retrieving the accent color.

Reply:
I suppose no OS lets you choose a transparent color so that might work `Color(0, 0, 0, 0)`.

Reply:
Updated to return `Color(0, 0, 0, 0)` color, and added note to the docs. A dedicated method is probably unnecessary.